### PR TITLE
Fixes for minimal running in 2019

### PR DESCRIPTION
**Pull request by tresoldi**

### CODE REVIEW

File: lingpy-server.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 cd app
-python server.py start
+python3 server.py start
```
Comment (LinguList) [inline lingpy-server.sh:2]: that depends on your working environment, mine is using python as basic code for python3, as are most archlinux systems,

Comment (LinguList) [reply lingpy-server.sh:2]: so better add another shell-script or an option

**Review by LinguList — APPROVED** [review]
apart from the shell thing, these changes are important (I have them in fact on my system here, just forgot to push)

File: lingpy-server.sh (hunk 1 above)
Comment (tresoldi) [reply lingpy-server.sh:2]: I think `python3` is supposed to be the canonical executable name, at least until January when Python2 will be deprecated.  But yes, minor issue.

**Pull request merged**